### PR TITLE
Hotfix/fix spinaker2.2

### PR DESCRIPTION
**Pull request by jpsnir**
Fixed the cmakelists for spinaker 2.2 installation. 

Need to change path for qt lib files for spinview to run.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -10,8 +10,8 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 ###
 #SET("OpenCV_DIR" "~/apps/opencv-2.4.13/")
 set(PROJECT_INCLUDE_DIR "${PROJECT_SOURCE_DIR}/include" CACHE PATH "Project Include Directory")
-set(SPINNAKER_INCLUDE_DIR "/usr/include/spinnaker" CACHE PATH "Spinnaker Include Directory")
-set(SPINNAKER_LIB_DIR "/usr/lib" CACHE PATH "Spinnaker Libs Directory")
+set(SPINNAKER_INCLUDE_DIR "/opt/spinnaker/include" CACHE PATH "Spinnaker Include Directory")
```
Comment (shahvi) [inline CMakeLists.txt:13]: This won't work.  This will break the code with older versions of spinnaker.  We need to have a way to let it still work with spinnaker 2.0 etc. as suggested in #105, we can to use find cmake OR add both the paths in there somehow.  @PushyamiKaveti  any ideas?

File: launch/acquisition_external_trigger.launch [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<launch>
```
Comment (shahvi) [inline launch/acquisition_external_trigger.launch:1]: This file doesn't belong in this pr.

File: params/external_trigger_params.yaml [hunk 3]
```diff
@@ -1,5 +1,9 @@
 cam_ids:
-- 17197559
+#- 17197559
```
Comment (shahvi) [inline params/external_trigger_params.yaml:2]: Remove this change from PR, should be local

File: params/test_params.yaml [hunk 4]
```diff
@@ -1,8 +1,9 @@
 cam_ids:
```
Comment (shahvi) [inline params/test_params.yaml:1]: Remove from this PR

File: src/capture.cpp [hunk 5]
```diff
@@ -275,15 +275,15 @@ void acquisition::Capture::load_cameras() {
     }
     ROS_ASSERT_MSG(cams.size(),"None of the connected cameras are in the config list!");
 
-    ROS_ASSERT_MSG(master_set,"The camera supposed to be the master isn't connected!");
+    //ROS_ASSERT_MSG(master_set,"Master cam error:The camera supposed to be the master isn't connected!");
```
Comment (shahvi) [inline src/capture.cpp:278]: These changes should be with the External trigger feature, not part of this PR.

**Review by shahvi — CHANGES_REQUESTED** [review]
Jagat most of the changes in this PR shouldn't be here.  Also the change in cmakelists will break the code for older versions.  We need to do it in a way that older spinnaker works too.

File: CMakeLists.txt (hunk 1 above)
Comment (PushyamiKaveti) [reply CMakeLists.txt:13]: Made the necessary changes to the CMakeLists.txt so that it works with both versions of spinnaker. This is fixed in branch hotfix/spinnaker_cmake

**Comment by shahvi** [comment]
This is now handled by FindCmake in PR #114
